study.md

```diff
@@ -85,7 +88,7 @@ Using Hash methods and string interpolation in Ruby, create a string using
 conspiracy theories.".
 
 ```ruby
-# your answer here
+"My name is #{[:dale.given_name]} and I'm an #{[:dale.occupation]} that enjoys #{[:dale.hobbies[1]]}."
```

Comment:
These will be interpreted as single-element arrays with the symbol `:dale` trying to invoke methods, which will cause an error.

`string = "My name is #{dale[:'given-name']} #{dale[:family_name]} and I'm an #{dale[:occupation]} that enjoys #{dale[:hobbies][1]}."`